index.html

```diff
@@ -7,70 +7,412 @@
                         </a>
                     </li>
                     <li>
                         <a href="#">
 
-                            <i class="fa fa-medium" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-medium" aria-hidden="true"></i>
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                            <i class="fa fa-github" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-github" aria-hidden="true"></i>
                             
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                            <i class="fa fa-angellist" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-angellist" aria-hidden="true"></i>
 
                         </a>
                     </li>
                 </ul>
                     <a href="#">
                         <i><img src="./assets/Disabled.png" alt=""></i>
                     </a>             
-        </footer>
+                </section>
     </section>
+
+    <main>
+        <section class="projects">
+            <header class="centered-column">
+                <h2 class="dark-blue">
+                    My Recent <br> Works
+                </h2>
+                <hr class="bg-green card hr">
+            </header>
+            <section class="work">
+
+                <section class="bordered-card">
+
+                    <section class="stretched-column card">
+                        <figure>
+
+                            <img src="../assets/place_holders/plaholder.png" class="bg-light-green place-holder" alt="">
+
+                        </figure>
+
+
+                        <section class="centered-column bg-white card">
+                            <header>
+                                <h3 class="minor-heading">
+                                    Multi-Post Stories <br> Gain+Glory
+                                </h3>
+                            </header>
+                            <ul class="tag-container">
+                                <li>
+                                    <p class="tag bg-light-green">Ruby on rails</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">css</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">JavaScript</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">html</p>
+                                </li>
+                            </ul>
+                            <a class="button bg-green white card" href="#" >See Project</a>
+                        </section>
```

Comment:
One of the major requirements is that you have button interactions whenever there is a need for it in your project. The `See Project` can be inside an `a` or `button` tag, but it should have `hover`, `active`, `focus` selectors whenever the button is hovered on, clicked, or focused on. You can read this [article](https://www.w3schools.com/cssref/sel_hover.asp) to be able to implement it.

- [ ] For your template, you can locate the colors to use for the interactions in this section:
![image](https://user-images.githubusercontent.com/77106962/143586236-ea5cdf6d-710a-45d2-a1ea-67c1d1c1be4f.png)
![image](https://user-images.githubusercontent.com/77106962/143586354-c5071056-3025-4f7f-bd98-303c101280bf.png)